e2e/process-services-cloud/start-process-cloud.e2e.ts

```diff
@@ -81,13 +80,6 @@ describe('Start Process', () => {
         startProcessPage.checkStartProcessButtonIsDisabled();
     });
 
-    it('[C291855] Should NOT be able to start a process without process model', () => {
```

Comment:
why this test has been removed?

Reply:
we don't have this case anymore because of: https://issues.alfresco.com/jira/browse/ADF-4295

Reply:
thanks